hub/load-test-grpc/README.md

```diff
@@ -1,19 +1,4 @@
-# Load test gRPC services with SLO validation
+# Documentation for `load-test-grpc`
 
-Iter8 experiment chart that enables load testing and validation of latency and error-related service level objectives (SLOs) for gRPC services.
-***
-
-## Examples
-
-The following `iter8 run` command will load test and validate the gRPC sample service with host `127.0.0.1:50051`, fully-qualified method name `helloworld.Greeter.SayHello`, and defined by the Protocol Buffer file located at the `protoURL`. The gRPC requests made by the Iter8 experiment will include `{'name': 'frodo'}` as the data, serialized in the protobuf format. The command also specifies that the error rate must be 0, the mean latency must be under 50 msec, the 90th percentile latency must be under 100 msec, and the 97.5th percentile latency must be under 200 msec.
-
-```shell
-iter8 run --set-string host="127.0.0.1:50051" \
-          --set-string call="helloworld.Greeter.SayHello" \
-          --set-string protoURL="https://raw.githubusercontent.com/grpc/grpc-go/master/examples/helloworld/helloworld/helloworld.proto" \
-          --set data.name="frodo" \
-          --set SLOs.error-rate=0 \
-          --set SLOs.latency/mean=50 \
-          --set SLOs.latency/p90=100 \
-          --set SLOs.latency/p'97\.5'=200
-```
+- [Overview](https://iter8.tools/0.8/tutorials/load-test-grpc/overview/)
+- [Unary gRPC service](https://iter8.tools/0.8/tutorials/load-test-grpc/unary/)
```

Comment:
These links have changed. Should be updated.

Reply:
Will fix once we cut a release. Note that mkdocs has the appropriate redirects for them already, so they will not be broken.

Reply:
i do see the redirects. for my own understanding, can you explain why waiting for release is a better option. i don't understand why wait ... and be forced to release 2x.

Reply:
Could we please handle the business of the links in a subsequent PR?

***

The repo structure, with code and docs together, and the practice of updating them both in the same PR, doesn't seem to be doing us favors.

Reply:
Updating the README links before the release would mean there is a period of time (between merge and release), when the projects README page has broken links.
***
I agree with thee point on `forced release 2x`. This is a bigger problem due to repo structure... which we will handle better in v0.9